receiver/awscontainerinsightreceiver/internal/stores/podstore.go

```diff
@@ -523,6 +527,35 @@ func (p *PodStore) addStatus(metric CIMetric, pod *corev1.Pod) {
 	}
 }
 
+func (p *PodStore) addPodStatusMetrics(metric CIMetric, pod *corev1.Pod) {
+	for _, metricName := range ci.PodPhaseMetricNames {
+		metric.AddField(metricName, 0)
+	}
+
+	statusMetricName, validStatus := ci.PodPhaseMetricNames[pod.Status.Phase]
+	if validStatus {
+		metric.AddField(statusMetricName, 1)
+	}
+}
+
+func (p *PodStore) addPodConditionMetrics(metric CIMetric, pod *corev1.Pod) {
+	for _, metricName := range ci.PodConditionMetricNames {
+		metric.AddField(metricName, 0)
+	}
+
+	for _, condition := range pod.Status.Conditions {
+		conditionValue, conditionIsUnKnown := strconv.ParseBool(string(condition.Status))
```

Comment:
We have consts for these as ConditionTrue, ConditionFalse and ConditionUnknown so would be better to not rely on conversions here.

Reply:
oh awesome love it

Reply:
done